packages/styles/scss/utilities/_tabs.scss

```diff
@@ -0,0 +1,23 @@
+// Tabs
+
+@use '../theme' as *;
+
```

Comment:
Since these are `Tabs` specific tokens, I think it would be best if we either put them at the top of `_tabs.scss`, or we created a `_vars.scss` inside of the `tabs` folder, and then use it in `_tabs.scss`.